universal-login-relayer/lib/core/utils/errors.ts

```diff
@@ -77,6 +77,13 @@ export class InvalidMessage extends NotFound {
   }
 }
 
+export class InvalidAddress extends NotFound {
```

Comment:
We should add new base exception `Unauthorised`, from which we should inherit, which should be handled with 401 http code in `errorHandler` here:
https://github.com/UniversalLogin/UniversalLoginSDK/blob/17c3cfd0291e965aed13692721ac8b11bcf39935/universal-login-relayer/lib/http/middlewares/errorHandler.ts#L34